roscpp_azure_iothub/CMakeLists.txt

```diff
@@ -50,7 +45,11 @@ find_package(azure_iot_sdks CONFIG)
 if(azure_iot_sdks_FOUND)
   message(STATUS "azure-iot-sdk-c is found - ${IOTHUB_CLIENT_INCLUDES}")
 elseif(EXISTS "/usr/include/azureiot")
-  set(IOTHUB_CLIENT_INCLUDES "/usr/include/azureiot" "/usr/include/azureiot/inc")
+  if(MSVC)
+    set(IOTHUB_CLIENT_INCLUDES "/usr/include/azureiot" "/usr/include/azureiot/inc")
+  else()
+    set(IOTHUB_CLIENT_INCLUDES "/usr/include/azureiot" "/usr/include/azureiot/inc" "parson")
```

Comment:
Parson is a submodule, so it should be brought in when you recursively clone the repo correct?
I think this code implies that you are using the system parson, which would have to be installed prior to building.